posts/postsapp/admin.py

```diff
@@ -0,0 +1,8 @@
+from django.contrib import admin
+
+# Register your models here.
+from .models import Topic, Article, Comment, UserTopic
```

Comment:
Не забувай чистити імпорти, не помилка.
```suggestion
from .models import Topic, Article, Comment
```